specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/AppConfiguration.json

```
@@ -50,6 +50,57 @@
           "type": "string"
         }
       }
+    },
+    "AppConfigurationSnapshotEventData": {
+      "description": "Schema of common properties of snapshot events",
+      "properties": {
+        "name": {
+          "description": "The name of snapshot.",
```

```suggestion
          "description": "The name of the snapshot.",
```

In our code emitters we append things like `that was created` so this ends up being `The name of snapshot that was created` which reads weird. Hopefully we can change this?

---

@billwert Yes, I have updated the description as you suggested, thanks for pointing it out